warehouse/accounts/forms.py

```diff
@@ -122,11 +123,14 @@ def validate_password(self, field):
                     raise wtforms.validators.ValidationError(
                         _("The password is invalid. Try again.")
                     )
-            except TooManyFailedLogins:
+            except TooManyFailedLogins as err:
                 raise wtforms.validators.ValidationError(
                     _(
                         "There have been too many unsuccessful login attempts. "
-                        "Try again later."
+                        "You have been locked out for {0}. "
+                        "Please try again later.".format(
+                            humanize.naturaldelta(err.resets_in.total_seconds())
+                        )
```

Comment:
It seems like this is the first time we're trying to pass in a value to a translated string like this. As-is, this produces a string that might get mis-translated and break the formatting. 

Instead, the `translate` function takes a `mapping` argument that should be used for this instead: https://docs.pylonsproject.org/projects/pyramid/en/latest/api/i18n.html#pyramid.i18n.Localizer.translate

We should be able to just do:

```suggestion
                        "You have been locked out for ${time}. "
                        "Please try again later.",
                        mapping={
                            'time': humanize.naturaldelta(err.resets_in.total_seconds()),
                        }
```

I'm not 100% sure that will work though, we might need to allow the `mapping` argument to be explicitly passed through to here: https://github.com/pypa/warehouse/blob/15cc65e57d20268ee2ce0a9ef4cdeb410d5069c7/warehouse/i18n/__init__.py#L98-L108

An explanation for how to do this is also missing from https://warehouse.readthedocs.io/translations/#passing-non-translatable-values-to-translated-strings. Once we figure this out, we should add it to the docs as well.